components/31-molecules/partner-block/partner-block.twig

```diff
@@ -0,0 +1,79 @@
+<div class="partner-block box-no-icon {{ modifier }}">
+  {% if title %}
+    {% include '@heading' with {
+      'type': 'h3',
+      'heading_text': title
+    } %}
+  {% endif %}
+
+  {% if description %}
+    {% include '@paragraph' with {
+      'text': description
+    } %}
+  {% endif %}
+
+
+  {% if single_partner %}
+    <div class="single-partner">
+      {% include '@image' with {
+        src: '/styleguide/img/stock/ocmw-logo.svg' | path,
+        alt_text: 'Partner logo',
```

Comment:
You're giving a bit of a bad example here.
Better to just use the company name (wouldn't include 'logo', it's implied).

Reply:
Oh okay sure. I thought I'd keep it generic.